flink-clients/src/main/java/org/apache/flink/client/CliFrontend.java

```diff
@@ -20,8 +20,6 @@
 
```

Comment:
The CliFrontend still contains YARN specific code like the YARN properties file parsing.

I wonder if there's a way to completely remove everything mentioning YARN from the CLI.